components/CodeEditor.vue

```diff
@@ -84,7 +99,13 @@ function onCloseAPIReferenceModal() {
       <div class="flex justify-end">
         <button
           type="submit"
-          class="h-10 px-6 font-semibold shadow bg-black text-white hover:bg-gray-800 transition"
+          class="h-10 px-6 font-semibold shadow text-white hover:bg-gray-800 transition"
+          :title="state.errorsInCode ? 'В коде допущена ошибка' : ''"
```

Comment:
Это должен быть тултип к заблокированной кнопке, а не текст самой кнопки. Можем вместе посмотреть, как сделать.

```suggestion
          :title="state.errorsInCode ? 'В коде допущена ошибка' : ''"
```

Reply:
Хотел через тултип, но лицезрев возможные ошибки, просто закинул в title, как описание при наведении

Reply:
@PlutoniumSoup, увидел, что `:title` кнопки – это не её текст. Так пойдёт и без специального тултипа. Тултип v1.